src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -293,6 +293,13 @@ func runClientProxy(cfg *config.Config) {
 	}
 
 	applyClientConfig(client, cfg)
+
+	// Only run analytics once on startup. It subscribes to IP discovery
+	// events from geolookup, so it needs to be subscribed here before
+	// the geolookup code executes.
+	analytics.Configure(cfg, version)
+	geolookup.Start()
```

Comment:
Just pulling these out -- an unrelated change but they really shouldn't be updated with every config update...